crates/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -805,7 +805,7 @@ pub(super) fn replace(s: &[Series], literal: bool, n: i64) -> PolarsResult<Serie
 #[cfg(feature = "string_from_radix")]
 pub(super) fn from_radix(s: &Series, radix: u32, strict: bool) -> PolarsResult<Series> {
```

Comment:
We should rename this method to match:
```suggestion
pub(super) fn to_integer(s: &Series, base: u32, strict: bool) -> PolarsResult<Series> {
```